### PR TITLE
chore(levelup): add type to CrudRepository

### PR DESCRIPTION
**Pull request by hulfyback**
set up the CrudRepository types in repository interfaces
(after it FTS-21 was pushed and merged, and there were models created)

<BadgleLevel, Long>
<Badge, Long>
<Person, Long>

### CODE REVIEW

File: src/main/java/com/greenfoxacademy/levelup/repository/IBadgeRepository.java [hunk 1]
```diff
@@ -6,6 +6,6 @@
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface IBadgeRepository extends CrudRepository<Badge, Level> {
+public interface IBadgeRepository extends CrudRepository<Badge, Long> {
```
Comment (rozanagy) [inline src/main/java/com/greenfoxacademy/levelup/repository/IBadgeRepository.java:9]: ok

File: src/main/java/com/greenfoxacademy/levelup/repository/IBadgeLevelRepository.java [hunk 2]
```diff
@@ -1,9 +1,10 @@
 package com.greenfoxacademy.levelup.repository;
 
+import com.greenfoxacademy.levelup.model.BadgeLevel;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface IBadgeLevelRepository extends CrudRepository {
+public interface IBadgeLevelRepository extends CrudRepository<BadgeLevel, Long> {
```
Comment (rozanagy) [inline src/main/java/com/greenfoxacademy/levelup/repository/IBadgeLevelRepository.java:8]: ok

File: src/main/java/com/greenfoxacademy/levelup/repository/IBadgeRepository.java [hunk 3]
```diff
@@ -1,9 +1,11 @@
 package com.greenfoxacademy.levelup.repository;
 
+import com.greenfoxacademy.levelup.model.Badge;
+import java.util.logging.Level;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface IBadgeRepository extends CrudRepository {
+public interface IBadgeRepository extends CrudRepository<Badge, Level> {
```
Comment (rozanagy) [inline src/main/java/com/greenfoxacademy/levelup/repository/IBadgeRepository.java]: correct Level to Long

File: src/main/java/com/greenfoxacademy/levelup/repository/IPersonRepository.java [hunk 4]
```diff
@@ -1,9 +1,10 @@
 package com.greenfoxacademy.levelup.repository;
 
+import com.greenfoxacademy.levelup.model.Person;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface IPersonRepository extends CrudRepository {
+public interface IPersonRepository extends CrudRepository<Person, Long> {
```
Comment (rozanagy) [inline src/main/java/com/greenfoxacademy/levelup/repository/IPersonRepository.java:8]: ok

**Review by bekobarna — APPROVED** [review]

**Pull request merged**